### PR TITLE
latest changes from private package

### PR DESCRIPTION
**Pull request by rcaplanshopify**
### Background

This PR releases several updates to the `unstable` packages.

If you've been tagged as a reviewer, **please add a changeset for your update(s).**

Here's a related [PR for the dev docs update](https://github.com/Shopify/shopify-dev/pull/37493) available to review on [spin](https://shopify-dev.delivery-target.rick-caplan.us.spin.dev/docs/api/checkout-ui-extensions/unstable/extension-targets-overview).

(Provide a link to any relevant issues AND provide a TLDR of the issue in this pull request)

### Solution

(Describe your solution, why this approach was chosen, and what the alternatives/impacts may be)

### 🎩

- ...

### Checklist

- [ ] I have :tophat:'d these changes
- [ ] I have updated relevant documentation

### CODE REVIEW

**Comment by lsit** [comment]
I added [this changeset](https://github.com/Shopify/ui-extensions/blob/unstable/.changeset/cuddly-spiders-tap.md) when I added the PaymentIcon to `unstable` ([PR](https://github.com/Shopify/ui-extensions/pull/1340)). Do I still need to do anything?

Also @jkeohan has a [PR](https://github.com/Shopify/ui-extensions/pull/1351) up to add ProductThumbnail to `unstable` with a [changeset](https://github.com/Shopify/ui-extensions/pull/1351/files#diff-97af5281f7c6e23a8b1a1ecb16e4f7381b948ce3a7313cdf1e8f35f2fe5663d5). He's still working through some comments there that weren't caught in the private package. Can the ProductThumbnail changes not be included here?

**Comment by rcaplanshopify** [comment]
> I added [this changeset](https://github.com/Shopify/ui-extensions/blob/unstable/.changeset/cuddly-spiders-tap.md) when I added the PaymentIcon to `unstable` ([PR](https://github.com/Shopify/ui-extensions/pull/1340)). Do I still need to do anything?
> 
> Also @jkeohan has a [PR](https://github.com/Shopify/ui-extensions/pull/1351) up to add ProductThumbnail to `unstable` with a [changeset](https://github.com/Shopify/ui-extensions/pull/1351/files#diff-97af5281f7c6e23a8b1a1ecb16e4f7381b948ce3a7313cdf1e8f35f2fe5663d5). He's still working through some comments there that weren't caught in the private package. Can the ProductThumbnail changes not be included here?

Thanks for the context @lsit! I don't think you need to do anything.

I will remove the ProductThumbnail-related changes from this branch this afternoon.

File: .changeset/green-bags-fetch.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/ui-extensions': minor
+'@shopify/ui-extensions-react': minor
+---
+
+add `purchase.checkout.payment-method-list.render-before` & `render-after` extension targets, rendered above and below the payment method list
```
Comment (jamesvidler) [inline .changeset/green-bags-fetch.md:6]:
```suggestion
Adds `purchase.checkout.payment-method-list.render-before` and `purchase.checkout.payment-method-list.render-before.render-after` extension targets, rendered above and below the payment method list.
```

**Review by elskhn — APPROVED** [review]
^ tophatted using same instructions, LGTM!

**Pull request merged**